src/domains/messages-common/03_database.tf

```diff
@@ -286,4 +286,30 @@ resource "azurerm_cosmosdb_sql_container" "remote_content_configuration" {
       path = "/\"_etag\"/?"
     }
   }
-}
\ No newline at end of file
+}
+
+resource "azurerm_cosmosdb_sql_container" "configuration" {
```

Comment:
Imho `configuration` is too simple name. I'd suggest `message-configuration` because this will be related to messages, and its id will be sent to CreateMessage.

Reply:
[6148dd3](https://github.com/pagopa/io-infra/pull/844/commits/6148dd31e7fce35bb9dfd273f3f4f333aed65597)